rails/benchmarks/support/benchmark_rails.rb

```diff
@@ -14,8 +14,10 @@ def rails(label=nil, time:, disable_gc: true, warmup: 3, &block)
         GC.enable
       end
 
-      report = Benchmark.ips(time, warmup, true) do |x|
-        x.report(label) { yield }
+      total_allocated_objects = get_total_allocated_objects do
```

Comment:
not sure if there is value in total allocated objects, per iteration is fine and it happens after warmup anyway.